docs/content/concepts/sui-move-concepts/packages/upgrade.mdx

```diff
@@ -236,7 +236,7 @@ This pattern forms the basis for upgradeable packages involving shared objects,
 To upgrade a package, your package must satisfy the following requirements:
 - You must have an `UpgradeTicket` for the package you want to upgrade. The Sui network issues `UpgradeCap`s when you publish a package, then you can issue `UpgradeTicket`s as the owner of that `UpgradeCap`. The Sui Client CLI handles this requirement automatically.
 - Your changes must be layout-compatible with the previous version.
-    - Existing `public` function signatures and struct layouts must remain the same.
+    - Existing `public` function signatures and struct layouts (including struct abilities) must remain the same.
```

Comment:
nit: could you actually break this line into two? I think function signature and struct layouts are two very distinct topics